gazebo/rendering/Camera.hh

```diff
@@ -393,6 +401,22 @@ namespace gazebo
       /// \return Far clip distance
       public: double FarClip() const;
 
+      /// \brief  Get the X focal length in pixels
+      /// \return X focal length in pixels
+      public: double ImageFocalLengthX() const;
+
+      /// \brief Get the Y focal length in pixels
+      /// \return Y focal length in pixels
+      public: double ImageFocalLengthY() const;
+
+      /// \brief Get the X principal point in pixels
+      /// \return X principal point in pixels
+      public: double ImageOpticalCentreX() const;
```

Comment:
I guess we probably want to stick to the US spelling "center" instead of the British "centre"

Reply:
sounds good.